### ISSUE TITLE
Train coherence term meta-latent (meta SAE layer, special SAE latent, or fine-tuned SAE) for coherence detection

### ISSUE THREAD

**Issue by speediedan** [speediedan/interpretune#110]
Explore several approaches for training a coherence term meta-latent, including meta SAE layers, special SAE latents, and finetuning SAEs with a coherence term included. Empirically evaluate which approach best captures coherence, considering both interpretability and generalizability.

(1) the coherence scalar can be a function of a subset of late saes concatenated. the meta sae layer (if used) would try to reproduce the coherence loss term (bounded 0 to 1) as a composition of a set of sae latents that best aligns with coherent vs non-coherent/ contradictory inputs (prob requiring fine-tuning or retraining of sae)
(2) could also just be a special sae latent that  decomposes the coherence prob using larger concept vector inputs or first try just final layer model activations directly 

is a scaler really the best way to capture coherence? only in a very limited sense, as input could be viewed as coherent to 
varying degrees from multiple perspectives.  

the second approach (2) might be easier to interpret  in some sense as it doesn't depend on upstream/lower order saes, 
but (1) may be easier in other senses, easier to start with using already trained saes and ultimately more powerful/generalizable.
 
A third method (3) that just trains/finetunes the usual saes with the coherence term included is probably the simplest and best place to start. Should explore all approaches empirically!!

finally, note using the absence of a ""doesn't make sense"" sae activation may make more sense than maximizing the coherence one

concept identification / decomposition (likely via a supporting network) may be crucial to success of either (coherence vs dissonance meta latent) approach"
